pkg/authorino/authorino.go

```diff
@@ -29,6 +29,7 @@ func Authorino(ns string) *authorinov1beta1.Authorino {
 					Enabled: &tlsEnabledTmp,
 				},
 			},
+			SecretLabelSelectors: "authorino.kuadrant.io/managed-by=authorino",
```

Comment:
Just for the record, now that `SecretLabelSelectors` is being set here, it can be anything that makes sense for Kuadrant, such as the previously intended `secret.kuadrant.io/managed-by=authorino`.

Whatever goes in this field defines what `Secret` resources Authorino will watch events about.

What goes in `authconfigs.spec.identity.apiKey.labelSelectors` (which is copied from `apiproducts.spec.securityScheme.apiKeyAuth.credential_source.labelSelectors`) defines what `Secret` resources will be cached as valid API keys to authenticate to the hosts of the `AuthConfig`, when the `AuthConfig` resource is reconciled.

Reply:
I just wanted to do it explicit for completeness

Thanks!